docs/src/main/tut/json.md

```diff
@@ -278,15 +278,16 @@ import org.http4s.client.dsl.io._
 import org.http4s.client.blaze._
 import cats.effect.IO
 import io.circe.generic.auto._
+import fs2.Stream
 
 // Decode the Hello response
-def helloClient(name: String): IO[Hello] = {
+def helloClient(name: String): Stream[IO, Hello] = {
```

Comment:
My comment got eaten.  I'd like to show here how one `Stream` client is used to make multiple calls into `IO` by passing the client to the thing that makes the call.  But this is good to Get It Green.